README.md

```diff
@@ -771,6 +800,56 @@ transaction = starkinfra.issuingtransaction.get("5155165527080960")
 print(transaction)
 ```
 
+### Issuing Enums
+
+#### Query MerchantCategories
+
+You can query multiple merchant categories according to filters.
+You may use MerchantCategories information to define specific IssuingRule's categories.
```

Comment:
You can query any merchant categories using this resource.
You may also use MerchantCategories to define specific IssuingRule category filters, either by types or by codes.